Rakefile

```diff
@@ -4,3 +4,5 @@
 require File.expand_path('../config/application', __FILE__)
 
 Rails.application.load_tasks
+
+load "pg_search/tasks.rb"
```

Comment:
Redundant, According to https://github.com/Casecommons/pg_search: 

"Other Active Record projects

In addition to installing and requiring the gem, you may want to include the PgSearch rake tasks in your Rakefile. This isn't necessary for Rails projects, which gain the Rake tasks via a Railtie."